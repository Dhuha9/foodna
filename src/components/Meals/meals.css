.available-meals{
    width: 1440px;
    height: 820px;
    background-color: #03B664;
}

.meals-title{
    color: aliceblue;
    padding-top: 40px;
    padding-bottom: 40px;
}

.meals-quote{
    color: aliceblue;
    font-size:large;
    padding-bottom: 50px;
}

.items{
    width: 1200px;
    display: flex;
    justify-content: center;
    align-items: center;
    justify-content:center;
    left: 0;

}

.card{
    height: 436px;
    width: 270px;
    top: 247px;
    top: 0;
    margin-left: 10px;
    margin-right: 10px;
    border: none;
}

.card-body{
    position: relative;
}

.Button{
    background-color: #E5E5E5;
    height: 60px;
    width: 100%;
    border: 0;
    border: none;
    color: black;
    position: absolute;
    bottom: 0;
    left: 0;
}

.Button:hover {
    color: #fff;
    background-color: #FFB701;
    border-color: #0062cc;
}

.card-img {
    width: 270px;
    height: 270px;
    object-fit: cover;
}

.card-title{
    position: absolute;

}

.card-text{
    position: absolute;
    bottom: 80px;
    left: 18px;
    color: gray;
}

.carousel-indicators{
    display: none;
}

