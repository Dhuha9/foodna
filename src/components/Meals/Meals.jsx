import React, { Component } from 'react';
import { Carousel, CarouselItem } from 'react-bootstrap';
import MealCard from '../MealCard/MealCard';
import '../Meals/meals.css';

export default class Meals extends Component {
  render() {
    return (
      <div>
        <div className="available-meals">
          <h1 className="meals-title">Available Meals Right Now</h1>
          <p className="meals-quote">
            `If you dont try this app, you wont become the hero you were meant
            to be`
          </p>
          <Carousel>
            <CarouselItem>
              <div className="items">
                <MealCard
                  img="tomato.jpg"
                  title="Canned Tomato Soup"
                  organization="Family Market"
                />
                <MealCard
                  img="donut.jpg"
                  title="Pastry Box"
                  organization="Chocolate Bar"
                />
                <MealCard
                  img="pantry.jpg"
                  title="Two days worth of pantry"
                  organization="7eleven"
                />
                <MealCard
                  img="rice.jpg"
                  title="Rice bags"
                  organization="Sprout Warehouse"
                />
              </div>
            </CarouselItem>
            <CarouselItem>
              <div className="items">
                <MealCard
                  img="tomato.jpg"
                  title="Canned Tomato Soup"
                  organization="Family Market"
                />
                <MealCard
                  img="donut.jpg"
                  title="Pastry Box"
                  organization="Chocolate Bar"
                />
                <MealCard
                  img="pantry.jpg"
                  title="Two days worth of pantry"
                  organization="7eleven"
                />
                <MealCard
                  img="rice.jpg"
                  title="Rice bags"
                  organization="Sprout Warehouse"
                />
              </div>
            </CarouselItem>
          </Carousel>
        </div>
      </div>
    );
  }
}
