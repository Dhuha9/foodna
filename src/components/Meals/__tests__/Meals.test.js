import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Meals from '../Meals';
it('renders apply now correctly', () => {
  const tree = renderer.create(<Meals />).toJSON();
  expect(tree).toMatchSnapshot();
});
