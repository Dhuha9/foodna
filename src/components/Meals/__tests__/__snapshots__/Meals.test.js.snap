// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders apply now correctly 1`] = `
<div>
  <div
    className="available-meals"
  >
    <h1
      className="meals-title"
    >
      Available Meals Right Now
    </h1>
    <p
      className="meals-quote"
    >
      If you dont try this app, you wont become the hero you were meant to be
    </p>
    <div
      className="sc-jSgupP ewJlEB rec rec-carousel-wrapper "
      style={Object {}}
    >
      <div
        className="sc-eCssSg galrjd rec rec-carousel"
        size={
          Object {
            "height": 0,
          }
        }
        style={
          Object {
            "height": 0,
          }
        }
      >
        <button
          className="sc-bdfBwQ iLquPE rec rec-arrow rec rec-arrow-left"
          disabled={true}
          onClick={[Function]}
          style={Object {}}
          tabIndex={0}
          type="button"
        >
          ❮
        </button>
        <div
          className="sc-dlfnbm cukhLH rec rec-slider-container"
        >
          <div
            className="sc-hKgILt kFXEwu rec rec-slider"
            style={
              Object {
                "left": "0px",
                "right": "auto",
                "top": "auto",
                "transition": "all 500ms ease",
              }
            }
          >
            <div
              className="rec rec-swipable"
              onMouseDown={[Function]}
              style={
                Object {
                  "display": "flex",
                  "flexDirection": "row",
                }
              }
            />
          </div>
        </div>
        <button
          className="sc-bdfBwQ iLquPE rec rec-arrow rec rec-arrow-right"
          disabled={true}
          onClick={[Function]}
          style={Object {}}
          tabIndex={0}
          type="button"
        >
          ❯
        </button>
      </div>
      <div
        className="sc-iBPRYJ bkVSTc rec rec-pagination"
      >
        <button
          className="sc-gKsewC cJeRRR rec rec-dot rec rec-dot_active"
          onClick={[Function]}
          tabIndex={-1}
          type="button"
        />
      </div>
    </div>
  </div>
</div>
`;
